decidim-initiatives/app/views/decidim/initiatives/initiatives/index.html.erb

```diff
@@ -1,3 +1,11 @@
+<% content_for :js_content do %>
+  <%= append_javascript_pack_tag "decidim_initiatives" %>
+<% end %>
+
+<% content_for :css_content do %>
+  <%= append_stylesheet_pack_tag "decidim_initiatives", media: "all" %>
+<% end %>
```

Comment:
@entantoencuanto the `content_for` is not required anymore, isn't it?

Reply:
Done!